src/main/kotlin/coffeebot/Main.kt

```diff
@@ -2,17 +2,21 @@ package coffeebot
 
 import coffeebot.processor.Offline
 import coffeebot.processor.Online
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.options.option
 
-fun main(args: Array<String>) {
-    if (args.size != 1) {
-        println("Expecting exactly 1 arg: offline|BOT_TOKEN")
-        return
-    }
+fun main(args: Array<String>) = Main().main(args)
+
+class Main : CliktCommand() {
+    val botToken by argument()
```

Comment:
Could you add help text for botToken too, that it either expects a Discord secret or the string offline.

Also, I think the README.md needs to be updated to take --botToken, rather than just a single unnamed arg